databroker/assets/base_registry.py

```diff
@@ -689,15 +689,28 @@ def safe_join(inp):
                                          cmd_kwargs=dict(shift=shift),
                                          cmd='shift_root')
 
-    def correct_root(self, resource, new_root, verify):
```

Comment:
How do you want this documented as an API change? (Sorry I don't know the workflow here)